src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreCommand.cs

```diff
@@ -114,6 +119,21 @@ public async Task<RestoreResult> ExecuteAsync(CancellationToken token)
             {
                 telemetry.TelemetryEvent.AddPiiData(ProjectFilePath, _request.Project.FilePath);
 
+                bool areNamespacesEnabled = _request?.PackageNameSpaces.AreNamespacesEnabled ?? false;
```

Comment:
nit: This method is kinda long, consider adding a method that populates the properties for example.

Reply:
Now instead of 3 only 1 `areNamespacesEnabled` property added so not long anymore.